YAFCui/Core/Structs.cs

```diff
@@ -35,6 +35,11 @@ public enum SchemeColor {
         GreyAlt,
         GreyText,
         GreyTextFaint,
+        // Magenta group (indicate overproduction)
+        Magenta,
```

Comment:
Out of the four new entries, only `Magenta` is used. Do we need the other three in there?

Reply:
Yes, the 4 entries are used for 'default', 'hover', 'text', 'disabled text' or something.

See https://github.com/have-fun-was-taken/yafc-ce/pull/70/files#diff-4de221ae69c7d3ff01036e4865ccd3ef4b7541e70c739e28e54ad1b0f686decaR35 for actual/correct details :smile:

Reply:
Technically, it might be the case that those enums do not to be defined expliitly (maybe :thinking: ), but for the other colors the same is done. So each color is always in a group of four 'sub colors'.

Reply:
Thank you for the explanation!